### PR TITLE
doc: Add Bluetooth changes to 1.10 release notes

### PR DESCRIPTION
**Pull request by jhedberg**
Signed-off-by: Johan Hedberg <johan.hedberg@intel.com>

### CODE REVIEW

File: doc/release-notes-1.10.rst [hunk 1]
```diff
@@ -67,7 +67,11 @@ Networking
 Bluetooth
 *********
 
-* details ...
+* Multiple qualification-related fixes for Bluetooth Mesh
+* Support for acting as Bluetooth Mesh Friend Node
```
Comment (carlescufi) [inline doc/release-notes-1.10.rst]: Support for the Bluetooth Mesh Friend Node role ?

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: doc/release-notes-1.10.rst (hunk 1 above)
Comment (jhedberg) [reply doc/release-notes-1.10.rst]: sure, fixed

File: doc/release-notes-1.10.rst [hunk 2]
```diff
@@ -67,7 +67,11 @@ Networking
 Bluetooth
 *********
 
-* details ...
+* Multiple qualification-related fixes for Bluetooth Mesh
+* Support for Bluetooth Mesh Friend Node role
+* Support for Bluetooth Mesh Foundation Client Models
+* New Bluetooth Mesh shell module and test application
+* Multiple stability fixes for the BLE Controller
```
Comment (carlescufi) [inline doc/release-notes-1.10.rst]: Can you add those two from 1.9.1?

Support for PA/LNA amplifiers in the BLE Controller
Support for additional VS commands in the BLE Controller

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: doc/release-notes-1.10.rst (hunk 2 above)
Comment (jhedberg) [reply doc/release-notes-1.10.rst]: done

**Review by carlescufi — APPROVED** [review]

**Review by dbkinder — APPROVED** [review]
+1

**Pull request merged**